src/legacy/core_plugins/status_page/index.js

```diff
@@ -21,15 +21,10 @@ export default function (kibana) {
   return new kibana.Plugin({
     uiExports: {
       app: {
-        title: 'Server Status',
+        title: 'Old Server Status',
         main: 'plugins/status_page/status_page',
         hidden: true,
-        url: '/status',
-      },
-      injectDefaultVars(server) {
-        return {
-          isStatusPageAnonymous: server.config().get('status.allowAnonymous'),
-        };
+        url: '/old-status',
```

Comment:
Is it possible to land on it?

Reply:
Yea, by manually typing the path in the browser, it unfortunately is. Not sure how to not break the legacy optimizer without exposing the app, and the `url` is mandatory...

Maybe I should change the path to something a little scarier, like `__deprecated__/status`?

Reply:
I'm fine to keep it. We are about to remove it soon.